spec/index.js

```diff
@@ -33,6 +33,12 @@ app.commandLine.appendSwitch('host-resolver-rules', [
   'MAP notfound.localhost2 ~NOTFOUND'
 ].join(', '));
 
+// Enable features required by tests.
+app.commandLine.appendSwitch('enable-features', [
+  // spec/api-web-frame-main-spec.ts
+  'DocumentPolicyIncludeJSCallStacksInCrashReports'
```

Comment:
Any reason to not enable this feature by default `DocumentPolicyIncludeJSCallStacksInCrashReports` in the runtime ? Users are still required to opt-in respective documents via the policy header, just wondering if there is necessity to control the feature behind two runtime feature flags. Maybe in our case just the document policy is sufficient, thoughts ?

Reply:
I didn't consider this given the early API availability in Chromium. It's estimated to be in origin trial until Chromium 132 and there hasn't been significant feedback by standards committees based on information provided in [the Chrome Status page.](https://chromestatus.com/feature/4731248572628992)

Assuming it's fully rolled out in Chromium 133, we could expect to not require the flag by Feb 4, 2025.

> Maybe in our case just the document policy is sufficient, thoughts?

I'm not against it if folks are okay with this.

Reply:
Sounds good, we can revisit this again.